ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -57,11 +57,11 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
     Future.sequence(Vector.fill(n)(allocateParty()))
 
   final def create[T <: Template[T]: ValueDecoder](template: Template[T])(party: Party)(
-      implicit context: LedgerTestContext): Future[Contract[T]] =
+      implicit context: LedgerTestContext): Future[(String, Contract[T])] =
```

Comment:
@stefanobaghino-da: I think it'd be overall cleaner to have a second method (like `createWithTransactionId`) to not have to ignore the transaction ID if it's not needed.

Reply:
I'm having a look precisely now on how to go about this.